package.json

```diff
@@ -2,15 +2,9 @@
   "name": "svelte-jester",
   "version": "2.3.2",
   "description": "A Jest transformer for Svelte - compile your components before importing them into tests",
-  "main": "dist/transformer.cjs",
+  "type": "module",
   "module": "dist/transformer.mjs",
-  "exports": {
```

Comment:
Why did you get rid of all the exports?

According to 
- https://nodejs.org/api/packages.html#main-entry-point-export
- https://nodejs.org/api/packages.html#conditional-exports

this is the prefeered way.

Reply:
I didn't get rid of them all, but reduced it down to a single export. It has to be that way to make ESM the default because if Jest finds anything that can be pulled in via `require` then it will use that. I actually got this idea from your PR :laughing: I was surprised by it at first, but after testing I saw you were right and it's actually the only way you can make it work.

Reply:
okay :)